packages/material-renderers/src/controls/MaterialDateControl.tsx

```diff
@@ -35,10 +35,15 @@ import { withJsonFormsControlProps } from '@jsonforms/react';
 import { FormHelperText, Hidden } from '@mui/material';
 import { DatePicker, LocalizationProvider } from '@mui/x-date-pickers';
 import { AdapterDayjs } from '@mui/x-date-pickers/AdapterDayjs';
-import { createOnChangeHandler, getData, useFocus } from '../util';
+import {
+  createOnBlurHandler,
+  createOnChangeHandler,
+  getData,
+  useFocus,
+} from '../util';
 
 export const MaterialDateControl = (props: ControlProps) => {
-  const [focused, onFocus, onBlur] = useFocus();
+  const [focused, onFocus] = useFocus();
```

Comment:
We lose the focus handling of `useFocus` here, i.e. it expects that `onBlur` is called at some point to remove the focus. Shouldn't we call the `onBlur` here in our custom `onBlur`?